package.json

```diff
@@ -4,60 +4,65 @@
   "license": "MIT",
   "angular-cli": {},
   "scripts": {
+    "ng": "ng",
     "start": "yarn build-themes && ng serve",
     "start-jit": "yarn build-themes && ng serve --aot=false",
-    "lint": "tslint -p src/tsconfig.json",
+    "start:prod": "yarn build-themes && ng serve --prod",
+    "lint": "tslint -p tsconfig.json",
     "test": "yarn build-themes && ng test",
     "pree2e": "webdriver-manager update",
-    "e2e": "protractor",
+    "e2e": "yarn build-themes && ng e2e",
+    "build": "yarn build-themes && ng build",
+    "build:highmem": "yarn build-themes && node --max_old_space_size=8192 ./node_modules/@angular/cli/bin/ng build",
```

Comment:
Is this something commonly required, or is there something special about our app? I'm surprised that it's necessary given the relatively simple nature of our docs app

Reply:
It's tracked here: https://github.com/angular/angular-cli/issues/13734. Based on this issue, it has become quite common and only minor progress has been made towards "solving it". In some cases, the CLI team has stated that increasing the available memory is expected.

I only ran into the issue when doing an AOT non-production build.

Reply:
I'm not sure if this would occur with Ivy as well since the Ivy build doesn't get that far into the build process, yet.

Reply:
It's also needed with Ivy.

Reply:
It looks like this is being addressed in PR https://github.com/angular/angular-cli/pull/15777.